src/containers/SetContainer/index.js

```diff
@@ -123,14 +123,15 @@ const SetContainer = props => {
           onClick={handleToggleSideSheet}
         />
       </TopBar>
+
       {!!set.id && (
         <>
           <ActionFab
             to={{
               pathname: `/simulation`,
               search: `?periods=${props.simulationPeriod}&sets=${snakeCase(
                 set.name,
-              )}&orgUnit=${set.simulationOrgUnit.id}`,
+              )}&orgUnit=${set?.simulationOrgUnit?.id}`,
```

Comment:
The `set` should always be something (because of `!!set.id` higher up, no? So the safe chaing is only needed on the `simulationOrgUnit` and there I think we should maybe not add a `orgUnit=` if we don't have the id.